SCons/Tool/msvc.xml

```diff
@@ -395,6 +395,7 @@ and extract the relevant variables from the result (typically
 <envar>%PATH%</envar>) for supplying to the build.
 This can be useful to force the use of a compiler version that
 &SCons; does not detect.
+<literal>MSVC_SCRIPT_ARGS</literal> is the arguments (whitespace separated) passed to this script.
```

Comment:
instead of "is the..."  use "allows specifying arguments (a string or a list of strings) to be passed to the script"